### PR TITLE
add logs

### PR DESCRIPTION
**Pull request by govindkothari**
Add logs to base class `ValueCheckOperator`

### CODE REVIEW

**Comment by govindkothari** [comment]
@ArgentFalcon @feng-tao

**Comment by feng-tao** [comment]
let's get it in next week. I am wondering are you trying to add some more logging or trying to debug certain issues?

**Comment by govindkothari** [comment]
I am trying to add more logging, since from the logs its not clear what is the output of the test check.

File: airflow/operators/check_operator.py [hunk 1]
```diff
@@ -137,6 +137,7 @@ def __init__(
     def execute(self, context=None):
         logging.info('Executing SQL check: ' + self.sql)
         records = self.get_db_hook().get_first(self.sql)
+        logging.info("Record: " + str(records))
```
Comment (ArgentFalcon) [inline airflow/operators/check_operator.py]: Is there any chance that this is a nontrivial amount of data we're shoving in the logs?

Comment (govindkothari) [reply airflow/operators/check_operator.py]: This operation is expected to return only one row

**Comment by govindkothari** [comment]
🔨

**Pull request merged**